Release Notes 2021-79 (#351)
JupiterOne/docs: release-notes/2021-79.md

# JupiterOne 2021.79 Release

September 9, 2021

## New Features and Improvements

- Introduced a new quick filter for accessing critical assets in the Assets app. Critical assets 
  is a class of assets for which you can create queries and alerts to quickly access the most crucial data.

>   Learn more about how to customize and use critical assets [here](https://github.com/JupiterOne/docs/blob/main/guides/asset-inventory-filters.md).

Reviewer: This description can be improved to be more compelling and better describe the "critical assets" feature. 

Also the link needs to be relative path, not a static link to the file on github, otherwise it won't be parsed correctly when published.

```
[here](../guides/asset-inventory-filters.md)
```

That said, the `asset-inventory-filters.md` file does NOT describe how to configure critical assets. Do we have a separate doc for it?

Author: @erkangz The PR https://github.com/JupiterOne/docs/pull/347 for the new doc is merged. So, it just has to be published to zendesk.

Reviewer 2: @erkangz @janettelynch what about:

Introduced a new quick filter for accessing critical assets in the Assets app. 

Critical assets provides a shortcut for you to more easily monitor important changes in your cloud environment and receive alerts for new issues such as security findings or changes in access controls around your critical assets. This allows you to quickly focus on and resolve the most important issues. 

Learn more about how to customize and configure your critical assets here: LINK

  operators `!=`, `!~=`, `!^=`, `!$=` when operating on a group of values.

  For example, 

  ```j1ql
  FIND jira_user WITH accountType != ('atlassian' OR 'app' OR 'customer')
  ```

  is the equivalent of:

  ```j1ql
  FIND jira_user WITH 
    accountType != 'atlassian' AND 
    accountType !=  'app' AND 
    accountType !=  'customer'
  ```

  Basically, J1QL interprets the above query to return all `jira_user` entities, 
  excluding those that have an `accountType` value of `atlassian` or `app` or `customer`. 

  !!! warning
      **This is a breaking change!** We have taken precautions to ensure saved questions 
      and queries are not inadvertently affected.  

      We have run maintenance jobs to update all saved queries in questions, 
      alerts, and Insights dashboard widgets.
      You do NOT have to make any changes manually in JupiterOne.
      
      However, if you have stored queries outside of your JupiterOne account 
      (such as in a custom script), please update those queries accordingly.   

  

- Updated the JupiterOne query language to enable relationship direction with the use 

>   of double arrows,`<<` and `>>`.

Reviewer: we should provide a quick example here and add link to the docs

Author: @erkangz Can you review my added example and let me know if it is valid?

- Added traversal filters for use with the `where` clause to support filtering via `AND` and 
  `OR` operators upon a single field, for example: `find User where User.id=(1 or 2 or 3)`. 

- Added search functionality to the Groups page in the Users & Access menu.

- Improved the text wrapping in the query search window.

- Query Anywhere now auto-focuses when opened so you can immediately start typing.

> - Improved messaging when bulk synchronization fails.

Reviewer: ```suggestion
- Improved error messages when a bulk synchronization fails.
```

Is this error message in the UI or API? Please add to clarify

Reviewer 2: This an error message in the UI.

# JupiterOne 2021.79 Release

September 9, 2021

## New Features and Improvements

> - Introduced a new quick filter for accessing critical assets in the Assets app.

Reviewer: ```suggestion
- Introduced a new filter to define **Critical Assets** in the Assets app, 
  and a **Smart Class** query shorthand.
```

>   Critical assets provides a shortcut for you and your SecOps teams to easily monitor important changes in your cloud environment
>   and receive alerts for new issues such as security findings or changes in access controls around your critical assets. 
>   This allows you to quickly focus on and resolve your most important security issues.

Reviewer: ```suggestion
    Critical Assets are defined using a preset combination of entities classes and attributes 
    (e.g. Hosts and Applications tagged as production). This configuration can be easily customized
    to match your specific definition of what critical assets mean for your organization.
    
    Learn more about how to customize and configure your critical assets 
    [here](../guides/critical-assets.md).
    
    Critical Assets can be reference in query via a specific "Smart Class" shorthand: `#CriticalAsset`. 
    
    This allows you to easily query and monitor important changes and receive alerts when a drift or
    problem occurs for any of the critical assets you care about. It allows you to focus and prioritize 
    on the most important risks.
    
    For example, to identify and alert on new critical assets added in the last 24 hours:
    
    `Find #CriticalAsset with _createdOn > date.now - 24hours`
    
    This is the initial release of this capability. We will be adding some pre-built alert rules and Insights
    dashboards for critical assets soon.
```

>   Learn more about how to customize and configure your critical assets [here](../guides/asset-inventory-filters.md).

Reviewer: this can be deleted if we use the suggestion above.
